eo-maven-plugin/src/test/java/org/eolang/maven/HomeTest.java

```diff
@@ -80,4 +81,16 @@ void existsInDirTest(@TempDir final Path temp) throws IOException {
             Matchers.is(true)
         );
     }
+
+    @Test
+    void existsInDirDifferentEncryptionTest(@TempDir final Path temp) throws IOException {
+        final String filename = "文件名.txt";
+        final byte[] bytes = filename.getBytes(StandardCharsets.UTF_16BE);
+        final String decoded = new String(bytes, StandardCharsets.UTF_16BE);
+        new Home(Paths.get("directory")).save("any content", temp.resolve(decoded));
+        MatcherAssert.assertThat(
+            new Home(Paths.get("directory")).exists(temp.resolve(filename)),
+            Matchers.is(true)
+        );
+    }
```

Comment:
@includealex Can you add a new test with this path: `EOorg/EOeolang/EOmath/EOnan$EOas_int$EO@` and check replaced name